roles/install-k8s/tasks/main.yml

```diff
@@ -14,6 +16,7 @@
 - name: Remove domainname from the hostname
   shell: |
     hostnamectl set-hostname $(hostname | cut -d "." -f1)
+  when: ansible_architecture == 'ppc64le'
```

Comment:
why this is not applicable for s390x platform?

Reply:
For s390x VSIs, domainname is not appended to the hostname.